docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/cinema_service
+      - my_media:/files/media
+    command: >
+      sh -c " python manage.py wait_for_db &&
+              python manage.py migrate &&
+              python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - test_host
```

Comment:
The `depends_on` directive ensures that the `test_host` service is started before `cinema_service`, but it does not wait for the database to be ready. Consider using a script or tool to check the database readiness before starting the application.